tests/foreman/ui/test_myaccount.py

```diff
@@ -15,24 +15,93 @@
 
 @Upstream: No
 """
+import random
 
+from fauxfactory import gen_string, gen_alpha
+from nailgun.entities import User
+from robottelo.constants import LANGUAGES
 from robottelo.decorators import stubbed, tier1
 from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators
+from robottelo.ui.locators import locators
+from robottelo.ui.session import Session
+
+
+def _valid_string_data(min_len=1, max_len=50):
+    """Generates valid string data for ui tests
+
+    :param min_len: min len for generated string
+    :param max_len: max len for generated string. Default is 50 because of
+    ui validation
+    :return: list of generated strings
+    """
+    return [
+        gen_string('alphanumeric', random.randint(min_len, max_len)),
+        gen_string('alpha', random.randint(min_len, max_len)),
+        gen_string('cjk', random.randint(min_len, max_len)),
+        gen_string('latin1', random.randint(min_len, max_len)),
+        gen_string('numeric', random.randint(min_len, max_len)),
+        gen_string('utf8', random.randint(min_len, max_len)),
+    ]
```

Comment:
Instead of recreating this `_valid_string_data` helper here, you can reuse the existing datafactory functions, one example is https://github.com/SatelliteQE/robottelo/blob/master/robottelo/datafactory.py#L36

to achieve the same as your function above

```python

from robotello.datafactory import generate_strings_list
_valid_string_data = generate_strings_list(min_length=3, max_length=30, exclude_types=['html'])
```